src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx

```diff
@@ -95,10 +95,10 @@ class EduBenefitsApp extends React.Component {
 
 function mapStateToProps(state, ownProps) {
   return {
-    pageState: state.uiState.pages,
+    pageState: state.form1990.uiState.pages,
```

Comment:
Is the idea that all education forms will have the same store, and within the store it will be divided between each form? This seems like another thing that would be nice if it were automatically configured.